titiler/api/endpoints/stac.py

```
@@ -446,10 +423,7 @@ async def stac_tilejson(
     kwargs.pop("maxzoom", None)
 
     if not expression and not assets:
-        raise HTTPException(
-            status_code=status.HTTP_403_FORBIDDEN,
-            detail="Expression or Assets HAVE to be set in the queryString.",
-        )
+        raise BadRequestError("Expression or Assets HAVE to be set in the queryString.")
```

Should we raise `InvalidBandName` here as rio-tiler does?